docs/sql/kill.txt

```diff
@@ -6,35 +6,34 @@ Kill Jobs
 =========
 
 Each executed sql statement results in a corresponding job.
-Jobs that are currently executing are logged in the system table ``sys.jobs``
+Jobs that are currently executed are logged in the ``sys.jobs`` system table
```

Comment:
use `listed` instead of `logged`
because a few lines later, you tell people to disable logging ;)